search/processor.hpp

```diff
@@ -88,9 +86,11 @@ class Processor : public my::Cancellable
   inline m2::PointD const & GetPosition() const { return m_position; }
 
   /// Suggestions language code, not the same as we use in mwm data
-  int8_t m_inputLocaleCode, m_currentLocaleCode;
+  int8_t m_inputLocaleCode;
+  int8_t m_currentLocaleCode;
```

Comment:
Could you please init |m_inputLocaleCode| and |m_currentLocaleCode|?

Reply:
`#include <cstdint>`?

Reply:
Done